### PR TITLE
chore: code quality cleanups

### PR DESCRIPTION
**Pull request by miketheman**
See each commit for specifics.

### CODE REVIEW

File: warehouse/accounts/views.py [hunk 1]
```diff
@@ -793,7 +794,18 @@ def _error(message):
...
+            "user",
+            {
+                "username": user.username,
+                "last_login": user.last_login,
+                "token_last_login": last_login,
+            },
+        )
+        sentry_sdk.capture_message(
+            "Password reset token used after user logged in",
+            level="warning",
```
Comment (di) [inline warehouse/accounts/views.py:807]: Should this include a fingerprint or is the context sufficient to report different messages per user (assuming we want that).

Comment (miketheman) [reply warehouse/accounts/views.py:807]: What kind of fingerprint are you thinking of?

Comment (di) [reply warehouse/accounts/views.py:807]: Like this: https://github.com/pypi/warehouse/blob/e7e9d4ccadc852d33b7c475eaa2f749086227a09/warehouse/oidc/models/_core.py#L178-L182

Comment (di) [reply warehouse/accounts/views.py:807]: Maybe unique on the username?

Comment (miketheman) [reply warehouse/accounts/views.py:807]: Oh, I see! I'll update the message to include username in it, and that ought to trigger an issue per username. No need to much with custom scope or fingerprinting for that.

**Pull request merged**